src/rdkafka_buf.h

```diff
@@ -836,6 +836,29 @@ struct rd_kafka_buf_s { /* rd_kafka_buf_t */
                 }                                                              \
         } while (0)
 
+/**
+ * @brief Read KIP-482 Tags at current position in the buffer using
+ *        the `read_tag` function receiving the `opaque' pointer.
+ */
+#define rd_kafka_buf_read_tags(rkbuf, read_tag, opaque)                        \
```

Comment:
Change opaque to `...` for flexibility
```suggestion
#define rd_kafka_buf_read_tags(rkbuf, read_tag, ...)                        \
```

Reply:
Updated